src/self-model.scm

```diff
@@ -321,7 +331,6 @@
 			(AbsentLink (EvaluationLink (PredicateNode "acked face")
 					(ListLink (VariableNode "$face-id"))))
 		)))
-
```

Comment:
The blank line removed here, and the blank lines added below, look like more random accidents with the editor. Please fix (unless you really really meant to do this).